src/LazyImage/tests/BlurHash/BlurHashTest.php

```diff
@@ -37,6 +40,54 @@ public function testEncode()
         );
     }
 
+    public function testEncodeWithCache()
+    {
+        $kernel = new class('test', true) extends TwigAppKernel {
+            public function registerContainerConfiguration(LoaderInterface $loader)
+            {
+                parent::registerContainerConfiguration($loader);
+
+                $loader->load(static function (ContainerBuilder $container) {
+                    $container->loadFromExtension('framework', [
+                        'cache' => [
+                            'pools' => [
+                                'cache.lazy_image' => [
+                                    'adapter' => 'cache.adapter.array',
+                                ],
+                            ],
+                        ],
+                    ]);
+
+                    $container->loadFromExtension('lazy_image', [
+                        'cache' => 'cache.lazy_image',
+                    ]);
+
+                    $container->setAlias('test.cache.lazy_image', 'cache.lazy_image')->setPublic(true);
+                });
+            }
+        };
+
+        $kernel->boot();
+        $container = $kernel->getContainer()->get('test.service_container');
+
+        /** @var BlurHashInterface $blurHash */
+        $blurHash = $container->get('test.lazy_image.blur_hash');
+
+        $cache = $container->get('test.cache.lazy_image');
+        static::assertInstanceOf(ArrayAdapter::class, $cache);
+        static::assertEmpty($cache->getValues());
+
+        $this->assertSame(
+            'L54ec*~q_3?bofoffQWB9F9FD%IU',
+            $blurHash->encode(__DIR__.'/../Fixtures/logo.png')
+        );
+
+        static::assertSame(
+            'L54ec*~q_3?bofoffQWB9F9FD%IU',
+            $cache->getItem(array_key_first($cache->getValues()))->get()
```

Comment:
The generated cache key was different between my machine and the CI, so I'm using `array_key_first($cache->getValues())` instead 🤷🏻

Reply:
Maybe would be enough to set `framework.prefix.cache_seed` to get stable namespace :)

Reply:
To be more clear:
- on my machine the cache key is `blurhash.70d3aaba5c301af7`
- but on the CI it is `blurhash.788394be043a8824` (see https://github.com/symfony/ux/actions/runs/8745742961/job/24001335802?pr=1755#step:7:108 & https://github.com/symfony/ux/actions/runs/8745742961/job/24001338516?pr=1755#step:7:73)

The different part is the result of `hash('xxh3', '...')` and is not related to the cache prefix seed 😕  

And today we've learned, the namespace (generated through `framework.cache.prefix`) is not even used when adapter is an `ArrayAdapter` (https://github.com/symfony/cache/blob/7.0/DependencyInjection/CachePoolPass.php#L118-L119) :D

Reply:
Well, in fact it's super simple, since we compute the cache key with the filename, here `__DIR__.'/../Fixtures/logo.png'`, paths are different between my machine and CI :) 

But I we should keep this behaviour as is, I don't think we want to manipulate the filename (from public directory or not, from a URL or not, ...).